server/middleware/searchHandler.js

```diff
@@ -37,11 +37,14 @@ async function handleSearch(req, res, next) {
         const workstackData = await Promise.all(response.data.stages
             .sort(sortObjectByProp(workstackCase => workstackCase.caseReference))
             .map(bindDisplayElements(fromStaticList)));
-        const { workstackColumns } = await req.listService.fetch('S_SYSTEM_CONFIGURATION');
+        const { workstackTypeColumns } = await req.listService.fetch('S_SYSTEM_CONFIGURATION');
+        // using DEFAULT columns for search
+        const workstackColumnsForSearch = workstackTypeColumns[0].workstackColumns;
```

Comment:
Good enough for now... would be nice we we had a dedicated 'SEARCH_WORKSTACK' or similar here. Even if the columns would be the same as 'DEFAULT' for now. 
Saying this, it's very easy to change and improve in the future. I like how it turned out here.